frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/overview/simulated-region-overview.component.html

```diff
@@ -10,6 +10,16 @@
                     />
                 </ng-template>
             </ng-container>
+            <ng-container ngbNavItem="patients">
+                <a ngbNavLink data-cy="transferPointPopupNamesNav">Patienten</a>
+                <ng-template ngbNavContent>
+                    <app-simulated-region-overview-patients-tab
+                        class="mh-100 d-block h-100"
+                        [simulatedRegion]="(simulatedRegion$ | async)!"
```

Comment:
In general, we decided against passing whole objects as input in favor of passing ids. As some of the other components are also passing objects, I'm fine with keeping it how it is.